tests/unit/processor/labeler/test_labeler_rule.py

```diff
@@ -181,24 +160,38 @@ def test_regex_matches_returns_false_for_non_matching_document(self):
         ]
 
         for document in non_matching_documents:
-            assert not self.regex_rule.matches(document)
+            assert not rule.matches(document)
 
     def test_complex_regex_matches_returns_true_for_matching_document(self):
-        assert self.complex_regex_rule.matches({"applyrule": "UPlo8888"})
-        assert self.complex_regex_rule.matches({"applyrule": "UPlo99999"})
-        assert self.complex_regex_rule.matches({"applyrule": "UPlo$$$$"})
-        assert self.complex_regex_rule.matches({"applyrule": "UP$$$$88"})
+        rule_definition = {
+            "filter": r'applyrule: "(?:(?=.*[a-z])(?:(?=.*[A-Z])(?=.*[\d\W])|(?=.*\W)(?=.*\d))|(?=.*\W)(?=.*[A-Z])(?=.*\d)).{8,}"',  # pylint: disable=line-too-long
```

Comment:
Do we need these complicate regex filter in these test? (same in the following test)

Reply:
no but I didn't write it and so I didn't change it in my refactoring